src/components/Editor.tsx

```diff
@@ -30,6 +35,7 @@ export const Editor = ({ design }: { design: Design | undefined }) => {
     const unlayer = emailEditorRef?.current;
 
     if (emailEditorLoaded) {
+      setElementRef && setElementRef(unlayer);
       unlayer?.loadDesign(design || emptyDesign);
```

Comment:
Klever approach with the `setElementRef`!

Since an external component will have access to `unlayer` (the element ref), what do you think about moving the load responsibility out of the Editor component?

```suggestion
      setElementRef && setElementRef(unlayer);
```